includes/admin/meta-boxes/views/html-order-download-permission.php

```diff
@@ -39,20 +39,19 @@
 							'email'         => urlencode( $download->get_user_email() ),
 							'key'           => $download->get_download_id(),
 						), trailingslashit( home_url() ) );
-
-						echo '<a href="' . esc_url( $download_link ) . '">' . esc_html( $file_count ) . '</a>';
 					?>
+					<a id="copy-download-link" class="button" href="<?php echo esc_url( $download_link ) ?>" data-tip="<?php esc_attr_e( 'Copied!', 'woocommerce' ); ?>" data-tip-failed="<?php esc_attr_e( 'Copying to clipboard failed. You should be able to right-click the button and copy.', 'woocommerce' ); ?>"><?php _e( 'Copy link', 'woocommerce' ); ?></a>
 				</td>
 				<td>
-					<label><?php esc_html_e( 'Downloads completed', 'woocommerce' ); ?></label>
+					<label><?php esc_html_e( 'Customer download report', 'woocommerce' ); ?></label>
```

Comment:
Should we use 'log' terminology?

Do these need `:` after each?

Reply:
There aren't any colons: http://cld.wthms.co/nTlv1P. Not sure on the terminology though - maybe half and half, log in the title & report in the button? Example in above screenshot.